src/index.ts

```diff
@@ -1,24 +1,28 @@
-import Promise from 'promise-polyfill';
+import PromisePolyfill from 'promise-polyfill';
 import fetchPonyfill from 'fetch-ponyfill';
+import { Stripe } from '@stripe/stripe-js';
 import {
   SubmissionData,
   SubmissionOptions,
   SubmissionBody,
   SubmissionResponse
 } from './forms';
-import { clientHeader, encode64 } from './utils';
+import { appendExtraData, clientHeader, encode64 } from './utils';
 import { Session } from './session';
 
 export interface Config {
   project?: string;
+  stripePromise?: Stripe;
 }
 
 export class Client {
   project: string | undefined;
+  stripePromise: Stripe | undefined;
   private session: Session | undefined;
```

Comment:
Not sure why sometimes we use `?` and sometimes `| undefined`. Feel like we should just standardize around `project?: string` rather than `project: string | undefined`

Reply:
`?` means it's optional, and using `| undefined` means it's required but can undefined.